flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/sink/BaseDeltaTaskWriter.java

```diff
@@ -66,6 +70,10 @@ RowDataWrapper wrapper() {
     return wrapper;
   }
 
+  RowDataWrapper wrapperDelete() {
+    return wrapperDelete;
+  }
```

Comment:
Can you revert the changes in all modules other than 1.14? That makes reviewing and updating for review comments much easier. Once this goes in we can backport to 1.12 and 1.13.

Reply:
Yeah agreed. I left a similar comment on another file because the changes were a bit much.

It keeps the discussion of a specific change all in one place.

Reply:
In this case, I believe that @hililiwei made the changes as existing tests might not pass in earlier versions of Flink.

But for something important, we should still keep the changes in one PR while reviewing. Otherwise it's difficult for others to review.